front_end/src/app/(campaigns-registration)/bridgewater/components/registration-status.tsx

```diff
@@ -0,0 +1,260 @@
+"use client";
+
+import React, { FC, PropsWithChildren, useState } from "react";
+import { BWRegisterButton, ChoicesButtons } from "./hero-section";
+import cn from "@/utils/cn";
+import { CurrentUser } from "@/types/users";
+import BaseModal from "@/components/base_modal";
+import {
+  RegistrationAndSignupForm,
+  RegistrationForm,
+} from "./registration-forms";
+import { CAMPAIGN_KEY } from "../constants";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCheckCircle } from "@fortawesome/free-solid-svg-icons";
+import { faXmarkCircle } from "@fortawesome/free-regular-svg-icons";
+import { useRouter } from "next/navigation";
+import { useModal } from "@/contexts/modal_context";
+import Link from "next/link";
```

Comment:
I see some ESLint errors here. You can configure your editor to apply eslint fixes on save, so fixing it would be less annoying.

Reply:
Will check that out